lib/stitches/add_csrf_null_session_generator.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails/generators'
+
+module Stitches
+  class AddCsrfNullSessionGenerator < Rails::Generators::Base
```

Comment:
Should we add something to the `README` about this generator?  I see [some similar things](https://github.com/stitchfix/stitches#upgrading-from-an-older-version) mentioned there.